recipes/ibis-mssql/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: ibis-mssql
+  version: 0.1.3
+
+source:
+  url: https://pypi.io/packages/source/i/ibis-mssql/ibis-mssql-0.1.3.tar.gz
```

Comment:
This should be templated on the version. `https://pypi.io/packages/source/i/ibis-mssql/ibis-mssql-{{ version }}.tar.gz`

Reply:
Resolved. Thanks!